Resources/Catalog/LcgFileCatalogClient.py

```diff
@@ -2,13 +2,13 @@
 
 """
 import DIRAC
-from DIRAC                                                    import S_OK, S_ERROR, gLogger, gConfig
-from DIRAC.Resources.Utilities                                import checkArgumentFormat
-from DIRAC.Resources.Catalog.FileCatalogueBase                import FileCatalogueBase
-from DIRAC.Core.Utilities.Time                                import fromEpoch
-from DIRAC.Core.Utilities.List                                import breakListIntoChunks
-from DIRAC.Core.Security.ProxyInfo                            import getProxyInfo, formatProxyInfoAsString
-from DIRAC.ConfigurationSystem.Client.Helpers.Registry        import getDNForUsername, getVOMSAttributeForGroup, getVOForGroup, getVOOption
+from DIRAC                                                 import S_OK, S_ERROR, gLogger, gConfig
+from DIRAC.DataManagementSystem.Utilities.CatalogUtilities import checkCatalogArguments
+from DIRAC.Core.Utilities.Time                             import fromEpoch
```

Comment:
If we want to get ride of this "system-like" module like Time and Os, we should avoid introducing new use of them :-) You can call directly fromtimestamp